src/System.Collections.Concurrent/tests/System.Collections.Concurrent.Tests.csproj

```diff
@@ -98,6 +98,8 @@
     <Compile Include="$(CommonTestPath)\System\Collections\IEnumerable.Generic.Serialization.Tests.cs">
       <Link>Common\System\Collections\IEnumerable.Generic.Serialization.Tests.cs</Link>
     </Compile>
+    <Compile Include="ConcurrentBagTests.netstandard1.7.cs" />
```

Comment:
I assume this file is testing APIs that are only for netcoreapp correct? In which case this should use that naming convention. Also this test project currently only has netstandard build configurations (https://github.com/dotnet/corefx/blob/master/src/System.Collections.Concurrent/tests/Configurations.props#L4). You will need to add a netcoreapp configuration to correctly build this. 

This might correctly compile today because we aren't correctly picking the netstanard targeting pack when building this but it is currently incorrect.